### PR TITLE
Updates based on 3.2.5 release.

### PR DESCRIPTION
**Pull request by ashfurrow**
This updates the changelog and bundle version numbers for a forthcoming 3.2.6 release.

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
 upcoming:
-    version: 3.2.5
+    version: 3.2.6
     details: Bug fixes
     dev:
       - Adds client metadata info to causality events - ash
+
+releases:
+  - version: 3.2.6
```
Comment (alloy) [inline CHANGELOG.yml]: Is this supposed to be `3.2.5`?

Comment (ashfurrow) [reply CHANGELOG.yml]: Mmm, yup.

**Comment by ashfurrow** [comment]
LGTM?

**Comment by alloy** [comment]
Yup 👍

**Pull request merged**